packages/ember-metal/lib/mixin.js

```diff
@@ -317,11 +317,11 @@ function mergeMixins(mixins, m, descs, values, base, keys) {
       if (currentMixin._without) { a_forEach.call(currentMixin._without, removeKeys); }
     }
   }
-}
+};
 
 var IS_BINDING = /^.+Binding$/;
 
-function detectBinding(obj, key, value, m) {
+var detectBinding = (obj, key, value, m) => {
```

Comment:
i would prefer this to remain as a `function` declaration, as it has different hoisting rules.